features/sqlite-extensions.mdx

```diff
@@ -0,0 +1,36 @@
+
+<Info>
+  If you're looking to enable vector extensions, you should instead consider
+  using the native [libSQL vector datatype](/features/ai-and-embeddings).
+</Info>
+
+## Preloaded Extensions
+
+The following extensions are always-on and available:
+
+| Extension                                                                     | Version  | Description                                                                 |
+| ----------------------------------------------------------------------------- | -------- | --------------------------------------------------------------------------- |
+| [JSON](https://www.sqlite.org/json1.html)                                     | Built-in | Work with JSON data in SQLite.                                              |
+| [FTS5](https://www.sqlite.org/fts5.html)                                      | Built-in | Full-text search and indexing.                                              |
+| [R\*Tree](https://www.sqlite.org/rtree.html)                                  | Built-in | Indexing and querying spatial data.                                         |
+| [SQLean Crypto](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/crypto.md) | 0.24.1   | Hashing, message digest, encoding, and decoding.                            |
+| [SQLean Fuzzy](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/fuzzy.md)   | 0.24.1   | Fuzzy string matching and phonetics. A fork of Spellfix1 with improvements. |
+| [SQLean Math](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/math.md)     | 0.24.1   | Advanced mathematical calculations.                                         |
+| [SQLean Stats](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/stats.md)   | 0.24.1   | Common statistical functions with SQLite.                                   |
+| [SQLean Text](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/text.md)     | 0.24.1   | String manipulation (reverse, split) with SQLite.                           |
+| [SQLean UUID](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/uuid.md)     | 0.24.1   | Limited support for RFC 4122 compliant UUIDs.                               |
+
+## Optional Extensions
+
+The following extensions are also enabled in addition to the preloaded extensions on databases using the Fly Cloud Provider (Pro & Enterprise).
+
+| Extension                                                                     | Version | Description                                  |
+| ----------------------------------------------------------------------------- | ------- | -------------------------------------------- |
+| [SQLean Regexp](https://github.com/nalgeon/sqlean/blob/0.24.1/docs/regexp.md) | 0.24.1  | Regular expressions with SQLite.             |
+| [sqlite-vec](https://github.com/asg017/sqlite-vec/tree/v0.1.1)                | 0.1.0   | The Vector Search successor to `sqlite-vss`. |
```

Comment:
v0.1.1 or v0.1.0?

In the platform code I see `ENV VEC_VERSION="0.1.3"` env var